docs/platform-services/threat-intelligence/index.md

```diff
@@ -0,0 +1,49 @@
+---
+slug: /platform-services/threat-intelligence
+title: Threat Intelligence
+sidebar_label: Threat Intelligence 
+description: Learn about Sumo Logic's threat intelligence capabilities.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Threat intelligence is information that helps you prevent or mitigate cyber attacks. 
```

Comment:
Since this is the parent Threat Intel page (likely the first thing that'll come up on a Google search), I'd revise this to include a sentence containing _Sumo Logic_ to help with SEO.

Reply:
Done.